src/utils/string/StringUtils.js

```diff
@@ -326,7 +326,9 @@ const getChannelListInfo = (channel, selfSignUserId, selfAnonUserId) => {
 
     channelName = helperGetMemberName(member);
 
-    channelImage = isAnonymous ? DEFAULT_PROFILE_PIC_PATH : member?.user?.image;
+    channelImage = isAnonymous
+      ? DEFAULT_PROFILE_PIC_PATH
+      : member?.user?.image || DEFAULT_PROFILE_PIC_PATH;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for handling cases where `member?.user?.image` might be undefined or null. The code now falls back to `DEFAULT_PROFILE_PIC_PATH` if this is the case, which is a good practice to ensure that there's always a valid image path.
```diff
-    channelImage = isAnonymous ? DEFAULT_PROFILE_PIC_PATH : member?.user?.image;
+    channelImage = isAnonymous
+      ? DEFAULT_PROFILE_PIC_PATH
+      : member?.user?.image || DEFAULT_PROFILE_PIC_PATH;
```
This change improves the robustness of the code by providing a default value when the expected one is not available.


<!-- This is an auto-generated comment by OSS CodeRabbit -->